### PR TITLE
Test PR heading

### PR DESCRIPTION
**Pull request by ivantha**
- A
- B

### CODE REVIEW

File: app/.gitignore [hunk 1]
```diff
@@ -1 +1,3 @@
-/build
\ No newline at end of file
+/build
```
Comment (hashini-m) [inline app/.gitignore:1]: What is this change?

File: README.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
+# Cookit App
+
```
Comment (hashini-m) [inline README.md:2]: Delete empty line

**Review by hashini-m — CHANGES_REQUESTED** [review]

File: app/.gitignore (hunk 1 above)
Comment (ivantha) [reply app/.gitignore:1]: Fixed.

**Review by hashini-m — APPROVED** [review]
LGTM :+1:

**Pull request merged**